_data-prepper/common-use-cases/codec-processor-combinations.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Codec processor combinations
+parent: Common use cases
+nav_order: 25
+---
+
+# Codec processor combinations
+
+At ingestion time, data received by the [`s3 source`]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/sources/s3/) can be parsed by codecs, such as [Apache Avro](https://avro.apache.org/docs/1.11.1/specification/). which will compress and decompress large sets of data in a certain format before ingestion through a Data Prepper pipelines [processor]({{site.url}}{{site.baseurl}}/data-prepper/pipelines/configuration/processors/processors/).
```

Comment:
> such as [Apache Avro]

I recommend starting with a codec like JSON over Avro. JSON is probably the most popular codec.